doc/users/next_whats_new/separated_hatchcolor.rst

```diff
@@ -0,0 +1,63 @@
+Separated ``hatchcolor`` from ``edgecolor``
+-------------------------------------------
+
+`~matplotlib.patches.Patch` gained a new *hatchcolor* parameter to explicitly
+control hatch colors. Previously, hatch colors were the same as edge colors,
+with a fallback to :rc:`hatch.color` if the patch did not have an edge color.
+
+Fallback Logic
+~~~~~~~~~~~~~~
+When the *hatchcolor* parameter is specified, it will be used for the hatch.
+If it is not specified, it will fallback to using :rc:`hatch.color`.
+
+The special value 'edge' takes over the patch edgecolor, with a fallback to
+:rc:`patch.edgecolor` if the patch edgecolor is 'none'.
+
+If the patch hatchcolor is 'edge' then hatchcolor matches edgecolor, that is, it will
+be updated if edgecolor is changed (for example: by calling *set_edgecolor()*).
+But if hatchcolor is explicitly set (for example: by calling *set_hatchcolor()*
+or by using *hatch.color* rcParam), it will not be updated when edgecolor changes.
```

Comment:
I find this confusing. Logically, `hatchcolor='edge'` just always uses the the edgecolor, but that's already expressed in the previous paragraph. I don't know what information is needed on top of that. Do we need this paragraph at all? It reads a bit like an implementation detail.

Reply:
I've unresolved this, because I don't see any action relating the paragraph or a comment why it should stay as is.

Reply:
I'm sorry, missed this 🤦‍♂️